src/UNIV2LPOracle.t.sol

```diff
@@ -794,7 +794,15 @@ contract UNIV2LPOracleTest is DSTest {
         assertTrue(thirdVal > secondVal);                               // Verify price of WBTC0ETH LP token increased after trade
     }
 
-    function test_stop() public {
+    function test_stop_start() public {
+        assertEq(uint256(daiEthLPOracle.stopped()), 0);
+        daiEthLPOracle.stop();
+        assertEq(uint256(daiEthLPOracle.stopped()), 1);
+        daiEthLPOracle.start();
+        assertEq(uint256(daiEthLPOracle.stopped()), 0);
+    }
```

Comment:
```suggestion
    function test_stop_start() public {
        (bytes32 prevVal,) = daiEthLPOracle.peep();
        assertEq(uint256(daiEthLPOracle.stopped()), 0);
        daiEthLPOracle.stop();
        assertEq(uint256(daiEthLPOracle.stopped()), 1);
        (bytes32 val, bool has) = daiEthLPOracle.peep();
        assertEq(uint256(prevVal), uint256(val));
        assertTrue(has);
        daiEthLPOracle.start();
        assertEq(uint256(daiEthLPOracle.stopped()), 0);
    }
```

I think I got this change correct in the browser, but I would show that the price and has do not change when stopped.

Reply:
I've improved the test expanding the checks and also some pre computing was needed in order to have prices and be able to read them.